sqlalchemy_jsonapi/unittests/test_declarative_serializer.py

```diff
@@ -0,0 +1,214 @@
+"""Tests for declarative JSONAPISerializer serialize method."""
+
+from sqlalchemy_jsonapi.unittests.utils import testcases
+from sqlalchemy_jsonapi.unittests import models
```

Comment:
I don't want to be importing existing test data for this new information. Test data should belong to a `TestCase` or a module of `TestCase`s. Otherwise, this gets unweildy *really* quick.  Except for certain special test resources (databases, factories, etc), we should be able to see all the test resources, setup and teardown through just scanning the file.